bot/resources/tags/windows-path.md

```diff
@@ -1,10 +1,10 @@
 **PATH on Windows**
 
-If you have installed Python but you forgot to check the *Add Python to PATH* option during the installation you may still be able to access your installation with ease.
+If you have installed Python, but you forgot to check the *Add Python to PATH* option during the installation, you may still be able to access your installation with ease.
 
-If you did not uncheck the option to install the Python launcher then you will find a `py` command on your system. If you want to be able to open your Python installation by running `python` then your best option is to re-install Python.
+If you did not uncheck the option to install the Python launcher then you will find a `py` command on your system. If you want to be able to open your Python installation by running `python`, then your best option is to re-install Python.
```

Comment:
I see this repeated a lot, I do not see how these changes lead to better grammar?

Reply:
As noted above, this is mostly just to add a natural pause in speech

Reply:
I disagree that this improves grammar but if this is a change you would like to make that is fine with me. I will mark all of the comments I made about the added commas as resolved.